proposals/Level-1.md

```diff
@@ -231,25 +212,27 @@ If the selected catch block does not throw an exception, it must yield the
 value(s) expected by the corresponding catching try block. This includes popping
 the caught exception.
 
-Note that a caught exception value can be rethrown using the `rethrow`
-instruction.
+Note that a caught exception can be rethrown using the `rethrow` instruction.
 
 ### Rethrowing an exception
 
-The `rethrow` instruction takes the exception value associated with the
-`except_ref` on top of the stack, and rethrows the exception. A rethrow has the
-same effect as a throw, other than an exception is not created. Rather, the
-referenced exception value on top of the stack is popped and then thrown.
+The `rethrow` instruction takes the exception associated with the `except_ref`
+on top of the stack, and rethrows the exception. A rethrow has the same effect
+as a throw, other than an exception is not created. Rather, the referenced
+exception on top of the stack is popped and then thrown.
 
 ### Exception data extraction
 
-The `if_except block` defines a conditional query of the exception value on top
-of the stack. The exception value is not popped when queried. The if_except
-block has two subblocks, the `then` and `else` subblocks, like that of an `if`
-block. The then block is a sequence of instructions following the `if_except`
-instruction. The else block is optional, and if it appears, it begins with the
-`else` instruction.  The scope of the if_except block is from the `if_except`
-instruction to the corresponding `end` instruction.
+The `if_except` block begins with an `if_except` instruction, and
+has two instruction blocks, defined by the `then` and `else` instructions
```

Comment:
There isn't really a `then` instruction.

Reply:
Can't we add one, to be symmetric with `if` and `else` instruction pair?